client/blocks/comments/post-comment.jsx

```diff
@@ -318,7 +319,13 @@ class PostComment extends PureComponent {
 		} else if ( commentAuthor.site_ID ) {
 			commentAuthorUrl = getStreamUrl( null, commentAuthor.site_ID );
 		} else {
-			commentAuthorUrl = commentAuthor?.URL;
+			const urlToCheck = commentAuthor?.URL;
+			if ( urlToCheck && isURL( urlToCheck ) ) {
+				const protocol = getProtocol( urlToCheck );
+				if ( protocol === 'http:' || protocol === 'https:' ) {
```

Comment:
Technically, the comment author can link to any valid protocol. However, I think it would be prudent to only allow links to HTTP protocols.

Reply:
I suppose we aren't limiting it right now, so no regression here... but wondering "should we" be linking to unsecure http anyways?

Reply:
I updated it to only allow HTTPS in 5839d836f5d24c26959e5c62c80c3f2e9b4957d2